api/src/main/java/org/pmiops/workbench/institution/InstitutionService.java

```diff
@@ -114,4 +114,11 @@ boolean validateInstitutionalEmail(
 
   /** Returns {@code true} if eRA commons is required for given tier. */
   boolean eRaRequiredForTier(Institution institution, String accessTierShortName);
+
+  /**
+   * Returns the access tiers that the user's institution allow the user to join.
+   *
+   * @return
+   */
+  List<String> getUserEligabledAccessTiers(DbUser user);
```

Comment:
spelling: Eligible

Reply:
please update throughout this PR

Reply:
Done. Thanks!

Reply:
Please remove the letter 'd' throughout -> Eligible, not Eligibled